scale-continuum/README.md

```diff
@@ -0,0 +1,6 @@
+# Ehcache 3 Scale Continuum
+
+This sample demonstrates how Ehcache helps scaling.
+
+Start the `org.terracotta.sample.Main` class with JDK 8+ and open a browser on http://localhost:4567 to access
```

Comment:
@aurbroszniowski  is correct: the `verify` phase does start the app. But it's certainly worth adding to the doc, so let me do that.